AppManager/app_manager_server.py

```diff
@@ -724,6 +903,11 @@ def delete(version_key, port):
   options.define('db_proxy', appscale_info.get_db_proxy())
   options.define('tq_proxy', appscale_info.get_tq_proxy())
 
+  machine_instances_node = '/appscale/instances_by_machine/{}'.format(
```

Comment:
where is instances_by_machine used?

Reply:
In a later branch, the autoscaler will have a watch on that node to help make assignment decisions. I can take it out for now though since nothing is using it at the moment.